"Status","Start Time","Stop Time","Duration in ms","Parent Suite","Suite","Sub Suite","Test Class","Test Method","Name","Description"
"passed","Thu Oct 28 04:32:54 UTC 2021","Thu Oct 28 04:32:55 UTC 2021","934","","org.entando.kubernetes.controller.plugin.FailureTests","","org.entando.kubernetes.controller.plugin.FailureTests","shouldFailWhenTheDatabaseCouldNotBeProvisioned","shouldFailWhenTheDatabaseCouldNotBeProvisioned()","Should fail when the database capability provisioning failed"
"passed","Thu Oct 28 04:32:49 UTC 2021","Thu Oct 28 04:32:54 UTC 2021","4553","","org.entando.kubernetes.controller.plugin.DeployEntandoPluginTest","","org.entando.kubernetes.controller.plugin.DeployEntandoPluginTest","shouldDeployPluginImageWithPostgresql","shouldDeployPluginImageWithPostgresql()","Should deploy a custom Plugin image with a PostgreSQL database"
"passed","Thu Oct 28 04:32:56 UTC 2021","Thu Oct 28 04:32:56 UTC 2021","502","","org.entando.kubernetes.controller.plugin.FailureTests","","org.entando.kubernetes.controller.plugin.FailureTests","shouldLeaveEarlierExceptionInTactButStillLogIt","shouldLeaveEarlierExceptionInTactButStillLogIt()","Should log an exception message occurring during the execution of the DeploymentProcessor and leave it attached to the EntandoPlugin.status"
"passed","Thu Oct 28 04:32:55 UTC 2021","Thu Oct 28 04:32:56 UTC 2021","567","","org.entando.kubernetes.controller.plugin.FailureTests","","org.entando.kubernetes.controller.plugin.FailureTests","shouldFailWhenTheSsoServiceCouldNotBeProvisioned","shouldFailWhenTheSsoServiceCouldNotBeProvisioned()","Should fail when the SSO capability provisioning failed"
